### DISCUSSION TITLE
[Status Bars] Alternate Colors (Accessibility)

### DISCUSSION THREAD

**Discussion by Patricol** [runelite/runelite#16636] [Ideas]
I have mild colorblindness, and it is extremely difficult for me to differentiate the color of a venomed HP bar from the background color/pattern visible when not at full health.

Example:
![](https://camo.githubusercontent.com/d72a95f3d48659bf3153d1b17682ac266a07483969f0574a1a017c5d7ef5624f/68747470733a2f2f692e696d6775722e636f6d2f66393535514e642e706e67)

The minimap HP orb is easier to see because of the gradient and higher contrast; so this is mainly an issue of contrast for the Status Bars plugin specifically.

I'd like to see an option to either change the color of the background portion (visible when not at full health) or change the color of the venomed HP bar.

Thank you.

**Comment by JoJoPhantom** [comment]
Very big agree,   i'm majorly green colorblind,  and also have trouble with reds and blues.  (Trouble all around)
i'd love to see color customization in many places.
